lib/dehinter/font.py

```diff
@@ -14,12 +14,121 @@
 
 import array
 import os
+import pprint
+import sys
 from typing import Union
 
 from fontTools import ttLib  # type: ignore
 
 from dehinter.bitops import clear_bit_k, is_bit_k_set
 
+# instantiate pretty printer
+pp = pprint.PrettyPrinter(indent=4)
+
+
+# ========================================================
+# Core dehinting routine
+# ========================================================
+def dehint(tt,
+           keep_cvar=False,
+           keep_cvt=False,
+           keep_fpgm=False,
+           keep_gasp=False,
+           keep_glyf=False,
+           keep_hdmx=False,
+           keep_head=False,
+           keep_ltsh=False,
+           keep_maxp=False,
+           keep_prep=False,
+           keep_ttfa=False,
+           keep_vdmx=False,
+           verbose=True):
+    def report_actions(table, has_table):
+        if not has_table:
+            print("[-] Removed {table} table")
```

Comment:
```suggestion
            print(f"[-] Removed {table} table")
```

Need an f-string here

Reply:
yeah, I just saw that right now :-P